docs/runtimes/configuring-job-resources.md

```diff
@@ -204,7 +205,7 @@ Pods (services, or jobs created by those services) can have priorities, which in
 scheduling: a lower priority pod can be evicted to allow scheduling of a higher priority pod. Pod priority is relevant for all pods created 
 by the service. For MLRun, it applies to the jobs created by MLRun. For Nuclio it applies to the pods of the Nuclio-created functions.
 
-Eviction uses these values to determine what to evict with conjunction to the pods priority [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption).
+Eviction uses these values in conjuction with pod priority to determine what to evict [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption).
```

Comment:
Eviction uses these values in conjunction with pod priority to determine what to evict

Reply:
Hi @jillnogold, not sure if your comment is meant to be actionable or just for your own notes, but that is was the text was changed to (green)

Reply:
Hi @bcrant - looks like twice I hit Comment instead of cancel.... Sorry for the confusion